libs-scala/resources/src/main/scala/com/digitalasset/resources/ResourceFactories.scala

```diff
@@ -101,12 +98,12 @@ final class ResourceFactories[Context: HasExecutionContext] {
     * @tparam U The return type.
     * @return A [[Resource]] with a sequence of the values of the sequenced [[Resource]]s as its underlying value.
     */
-  def sequence[T, C[X] <: Traversable[X], U](seq: C[R[T]])(
```

Comment:
@S11001001 @SamirTalwar-DA My `CanBuildFrom` foo is very weak so please review this closely.

Reply:
It looks fine for me, and the tests should fail if it's wrong. I can't say for sure though because I haven't used 2.13's `Factory`.